src/Serilog.Sinks.RollingFile/project.json

```diff
@@ -9,7 +9,7 @@
     "iconUrl": "http://serilog.net/images/serilog-sink-nuget.png"
   },
   "dependencies": {
-    "Serilog.Sinks.File": "3.1.0"
+    "Serilog.Sinks.File": "3.2.0-dev-00762"
   },
```

Comment:
There will be a problem when you merge this into the `master`. "Classic" projects will not be able to install stable version of this package while it references a pre-release version of `Serilog.Sinks.File` package. I think the only way to install it will be with the `-Pre` switch